### PR TITLE
Add version suffix v1.0.0 to all default ClusterClass names

### PR DESCRIPTION
**Pull request by DanielXiao**
### What this PR does / why we need it
According to ClusterClass best practice, when we make any changes on it or its reference templates, we 'd better make a copy of all and rename them. Given multiple versions of the default ClusterClasses will coexist in the same management cluster, we add a version suffix in their names to identify them and avoid package conflicts.

There is no time to separate ClusterClass packages from the TKG meta package in Glasgow, we decided to hard code version v1.0.0 for now and will start to package them individually from Halifax.
### Which issue(s) this PR fixes

Fixes #

### Release note

```release-note

```

### CODE REVIEW

File: providers/infrastructure-aws/v2.0.0-beta.1/cconly/base.yaml [hunk 1]
```diff
@@ -1015,22 +1019,22 @@ spec:
...
 metadata:
-  name: aws-cluster-template
+  name: #@ "tkg-aws-default-{}-cluster".format(data.values.AWS_CLUSTER_CLASS_VERSION)
   namespace: #@ data.values.NAMESPACE
 spec:
   template:
     spec:
-      region: "${AWS_REGION}"
-      sshKeyName: "${AWS_SSH_KEY_NAME}"
+      region: #@ data.values.AWS_REGION
```
Comment (ydp) [inline providers/infrastructure-aws/v2.0.0-beta.1/cconly/base.yaml:1027]: I had a confusion for a long time about the difference between environment variable (by export) and config variable (in config file), when should we use environment variable and when should use config variable, or they are equivalent through some cli internal Golang code (uncertain...)? Thus, does this kind of change break user behavior?

Comment (ydp) [reply providers/infrastructure-aws/v2.0.0-beta.1/cconly/base.yaml:1027]: I remember a (uncertain) time ago when deleting AWS management cluster, AWS_REGION is necessary to be present as environment, not sure what's the current status.

Comment (DanielXiao) [reply providers/infrastructure-aws/v2.0.0-beta.1/cconly/base.yaml:1027]: Environment variable can not be interpolated by our ytt processor, you will see them still as ${AWS_REGION} after it is rendered

**Comment by zhouke1991** [comment]
LGTM

**Review by zhouke1991 — APPROVED** [review]

File: packages/tkg-clusterclass-aws/bundle/config/upstream/base.yaml [hunk 2]
```diff
@@ -5,22 +5,23 @@
 apiVersion: cluster.x-k8s.io/v1beta1
 kind: ClusterClass
 metadata:
-  name: tkg-aws-default
```
Comment (jeffwubj) [inline packages/tkg-clusterclass-aws/bundle/config/upstream/base.yaml:8]: modify it in overlay.yaml should be an option as well ?

**Review by 12345lcr — APPROVED** [review]
LGTM

File: packages/tkg-clusterclass-aws/bundle/config/upstream/base.yaml (hunk 2 above)
Comment (DanielXiao) [reply packages/tkg-clusterclass-aws/bundle/config/upstream/base.yaml:8]: Yes but overlay is usually for more complicated cases

**Review by wjun — APPROVED** [review]
LGTM.

**Pull request merged**